corehq/apps/sms/static/sms/js/compose.js

```diff
@@ -1,4 +1,10 @@
-hqDefine("sms/js/compose", function() {
+hqDefine("sms/js/compose",[
+    "jquery",
+    "hqwebapp/js/initial_page_data",
+    'jquery-ui/ui/sortable',
+    "hqwebapp/js/bootstrap-multi-typeahead",
```

Comment:
This file needs to be migrated to requirejs - wrapped in an hqDefine module and the global `RegExp.escape` removed.

Reply:
[docs on hqDefine](https://github.com/dimagi/js-guide/blob/master/code-organization.md#hqdefine) in case useful

Reply:
migrated it

Reply:
Nice, one small change: the main function doesn't need to be wrapped in `(function() { ... })()` anymore. That was likely there just to provide some encapsulation, but the hqDefine wrapper does the same thing.

Reply:
Cool, Done